### PR TITLE
Refresh connector permissions on save

### PR DESCRIPTION
**Pull request by lasryaric**
Fixes: https://github.com/dust-tt/tasks/issues/154

### CODE REVIEW

**Review by fontanierh — DISMISSED** [review]
Sounds good. Do we prefer to keep the `source` param then ?

**Comment by lasryaric** [comment]
@fontanierh remove the source params.

File: front/components/ConnectorPermissionsModal.tsx [hunk 1]
```diff
@@ -126,7 +126,11 @@ export default function ConnectorPermissionsModal({
         }
 
         await mutate(
-          `/api/w/${owner.sId}/data_sources/${dataSource.name}/managed/permissions`
+          (key) =>
+            typeof key === "string" &&
```
Comment (fontanierh) [inline front/components/ConnectorPermissionsModal.tsx:130]: do we still need the key thing ?

Comment (fontanierh) [reply front/components/ConnectorPermissionsModal.tsx:130]: Oh yeah for parents ?

**Review by fontanierh — APPROVED** [review]

File: front/components/ConnectorPermissionsModal.tsx (hunk 1 above)
Comment (lasryaric) [reply front/components/ConnectorPermissionsModal.tsx:130]: yes we still need it. It is here to invalidate all kids of this url if we see it as a tree with each letter being a kids.

**Pull request merged**